daemons/fenced/pacemaker-fenced.h

```diff
@@ -7,6 +7,8 @@
 
 #include <crm/common/mainloop.h>
 
+#define MAX_STONITH_HISTORY 100
```

Comment:
I'm not too concerned about the exact number here, because the logs serve as an alternate record, and because if stonith is having a high number of repeated failures, they are likely identical, and the user will likely need to look at the logs to figure out what is going on anyway.

I'm guessing each record averages around 60 bytes, so this would take up about 6K. I think we'd be fine with anything from 100-500.

Reply:
With the xml-overhead + possibly long node-names it might be a few 100 bytes per entry but as we have cib-messages >100K that still shouldn't matter.